packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -160,10 +160,10 @@ export default function ProjectNav({
   data,
   constructProjectURL,
   urls,
-  alerts,
   activeProduct,
   onProjectChange,
   hosts,
+  alerts = 0,
```

Comment:
Why not allow this to be undefined?

Reply:
harry suggested this, which allows us to conditionally render alerts/the badge based on whether or not alerts is > 0